quickstarts/scanned-events-query-governance/config.yml

```diff
@@ -0,0 +1,44 @@
+slug: query-governance
+
+title: Query Governance
+
+description: |
+  ## Quickstart for Query Governance
+  
+  This dashboard, being interactive and real-time, will provide a holistic view to the users on the volume of scanned events by queries and sources, thus enabling them to optimize resource utilization and improve query performance.
+
+  This quickstart includes dashboards for the following scanned events charts:
+  - General Scanned Events (Query Volume)
+  - Volume by Account
+  - Volume by Source
+  - Volume by API Key
+  - Volume by User
+  - Volume by Individual Dashboards
+
+
+summary: |
+  Dissect the query volume by source.  identify any dashboards that are generating a large number of queries, possibly indicating a need for optimization.
+
+level: Community
+icon: logo.png
+
+authors:
+  - Carolina Rotstein
+
+keywords:
+  - Secanned Events
+  - NRDB
+  - Query
+  - Governance
+
+documentation:  
+  - name: New Relic NRDB
+    url: https://docs.newrelic.com/docs/data-apis/get-started/nrdb-horsepower-under-hood/
+    description: Learn more about NRDB returns results for queries of all sizes astonishingly fast. To do this, we use parallel processing at massive scale.
+
+
+dataSourceIds:
+  - scanned-events-query-governance
```

Comment:
Can you please create a folder under "data-sources" directory.
Here is the reference: https://github.com/newrelic/newrelic-quickstarts/tree/main/data-sources/aerospike